utilities_common/chassis.py

```diff
@@ -16,3 +16,32 @@ def get_chassis_local_interfaces():
                 lst = data[1].split(",")
                 return lst
     return lst
+
+
+def is_smartswitch():
```

Comment:
I have added unit tests in my https://github.com/sonic-net/sonic-utilities/pull/3566, Please add unit tests for other get_all_dpu_options and get_all_options?

Reply:
@vvolam I can but it will be duplication of effort. I have the tests here. You can combine these with your commit or I'll do it after your commit.  
    def test_get_all_dpus(self, mock_device_info):
        # Mock get_dpu_list to return a list of DPUs
        mock_device_info.get_dpu_list = mock.Mock(return_value=['DPU0', 'DPU1'])
        assert chassis.get_all_dpus() == ['DPU0', 'DPU1']

        # Mock get_dpu_list to raise an exception
        mock_device_info.get_dpu_list = mock.Mock(side_effect=Exception("Error"))
        assert chassis.get_all_dpus() == []

    def test_get_all_dpu_options(self, mock_device_info):
        # Mock get_dpu_list to return a list of DPUs
        mock_device_info.get_dpu_list = mock.Mock(return_value=['DPU0', 'DPU1'])
        assert chassis.get_all_dpu_options() == ['DPU0', 'DPU1', 'all']

        # Mock get_dpu_list to raise an exception
        mock_device_info.get_dpu_list = mock.Mock(side_effect=Exception("Error"))
        assert chassis.get_all_dpu_options() == ['all']